molgenis-jobs/src/main/java/org/molgenis/data/jobs/Job.java

```diff
@@ -3,13 +3,14 @@
 /**
  * Interface for molgenis jobs.
  */
-public interface Job<Result>
+@FunctionalInterface
+public interface Job<T>
```

Comment:
Sonatype says you should give your type parameters single letter names, so changing that here.